src/proposals/20251002/SparkEthereum_20251002.t.sol

```diff
@@ -0,0 +1,349 @@
+    function OPERATOR_NETWORK_OPT_IN_SERVICE() external view returns (address);
+}
+
+interface ISparkVaultV2 {
+    function asset() external view returns (address);
+    function DEFAULT_ADMIN_ROLE() external view returns (bytes32);
+    function depositCap() external view returns (uint256);
+    function hasRole(bytes32 role, address account) external view returns (bool);
+    function SETTER_ROLE() external view returns (bytes32);
+    function minVsr() external view returns (uint256);
+    function maxVsr() external view returns (uint256);
+    function TAKER_ROLE() external view returns (bytes32);
+}
+
+contract SparkEthereum_20251002Test is SparkTestBase {
+
+    uint256 internal constant FIVE_PCT_APY = 1.000000001547125957863212448e27;
+    uint256 internal constant TEN_PCT_APY  = 1.000000003022265980097387650e27;
+
+    address internal constant GROVE_SUBDAO_PROXY = 0x1369f7b2b38c76B6478c0f0E66D94923421891Ba;
+    address internal constant PYUSD              = 0x6c3ea9036406852006290770BEdFcAbA0e23A0e8;
+    address internal constant SYRUP              = 0x643C4E15d7d62Ad0aBeC4a9BD4b001aA3Ef52d66;
+
+    address internal constant PT_USDE_27NOV2025             = 0x62C6E813b9589C3631Ba0Cdb013acdB8544038B7;
+    address internal constant PT_USDE_27NOV2025_PRICE_FEED  = 0x52A34E1D7Cb12c70DaF0e8bdeb91E1d02deEf97d;
+
+    uint256 internal constant AMOUNT_TO_GROVE            = 1_031_866e18;
+    uint256 internal constant AMOUNT_TO_SPARK_FOUNDATION = 1_100_000e18;
+
+    // Symbiotic addresses
+    address constant NETWORK_DELEGATOR = 0x2C5bF9E8e16716A410644d6b4979d74c1951952d;
+    address constant NETWORK_REGISTRY  = 0xC773b1011461e7314CF05f97d95aa8e92C1Fd8aA;
+    address constant OPERATOR_REGISTRY = 0xAd817a6Bc954F678451A71363f04150FDD81Af9F;
+    address constant RESET_HOOK        = 0xC3B87BbE976f5Bfe4Dc4992ae4e22263Df15ccBE;
+    address constant STAKED_SPK_VAULT  = 0xc6132FAF04627c8d05d6E759FAbB331Ef2D8F8fD;
+
+    constructor() {
+        id = "20251002";
+    }
+
+    function setUp() public {
+        setupDomains("2025-09-24T13:31:00Z");
+
+        deployPayloads();
+
+        // chainData[ChainIdUtils.Ethereum()].payload = 0x7B28F4Bdd7208fe80916EBC58611Eb72Fb6A09Ed;
+    }
+
+    function test_ETHEREUM_morpho_increasePTUSDE27NovSupplyCap() public onChain(ChainIdUtils.Ethereum()) {
+        _testMorphoCapUpdate({
+            vault: Ethereum.MORPHO_VAULT_USDS,
+            config: MarketParams({
+                loanToken:       Ethereum.USDS,
+                collateralToken: PT_USDE_27NOV2025,
+                oracle:          PT_USDE_27NOV2025_PRICE_FEED,
+                irm:             Ethereum.MORPHO_DEFAULT_IRM,
+                lltv:            0.915e18
+            }),
+            currentCap: 500_000_000e18,
+            newCap:     1_000_000_000e18
+        });
+    }
+
+    function test_ETHEREUM_sparkLend_reserveFactor() public onChain(ChainIdUtils.Ethereum()) {
```

Comment:
I would call this `test_ETHEREUM_sparkLend_increaseLbtcSupplyCapAutomatorParameters`